frontend/views/components/modal/README.md

```diff
@@ -74,3 +74,28 @@ A full-width Modal with just `transition` as wrapper. Note that this Modal has "
 ---
 
 Caught a mistake or want to contribute to the documentation? Edit this page on GitHub!
+
+### The `Prompt`
+To call a prompt with yes / no question we need to call sbp with this format:
+
+```js
+sbp('okTurtles.events/emit', OPEN_MODAL, 'Prompt', null, {
+  question: L('Would you like fresh tacos?'),
+  heading: L('Update info'),
+  yesButton: L('Yes'),
+  noButton: L('No')
+})
+```
+
+Following by a listener to the response:
+
+```js
+sbp('okTurtles.events/on', MODAL_RESPONSE, function (response) {
+  sbp('okTurtles.events/off', MODAL_RESPONSE)
+  if (response) {
+    console.log('Yes')
+  } else {
+    console.log('No')
+  }
+})
+```
```

Comment:
@pieer So this involves 3 calls to sbp, and this API is much more difficult to use than the one [that was requested](https://github.com/okTurtles/group-income/pull/1168#pullrequestreview-902530557).

The API that was requested is based on the browser's built-in `window.prompt()` - a single function call. Can you please add that API instead of this one?

Also, do you think you could add an example to the DesignSystem? E.g. a button, when clicked, that brings up the prompt?